### ISSUE TITLE
[public-1.10.0] MapHack: Can't find certain datadesc fields

### ISSUE THREAD

**Issue by dysphie** [nmrih/source-game#1091]
The following snippet fails to run:

```cpp
$edit_field
{
	"id"		"a" // func_brush		
	"fieldname"	"m_vecMins"
	"value"		"10 -100 30"
}
```

with warning:

>  MapHack WARNING: Couldn't find an entity datadesc field named "m_vecMins" (a)

but the data exists and Sourcemod finds it just fine:

```cpp
int ent = -1;
while ((ent = FindEntityByClassname(ent, "func_brush")) != -1)
{
	float min[3];
	GetEntPropVector(ent, Prop_Send, "m_vecMins", min);
	PrintToServer("m_vecMins: %.1f, %.1f, %.1f" , min[0], min[1], min[2]);
}
```

![sm](https://user-images.githubusercontent.com/11559683/64830986-e7673200-d5a9-11e9-8a94-7cd9bf2e4fb6.png)

It can also set it successfully.
I don't know if this is a bug, limitation or a user error, to be honest.

**Comment by dysphie** [comment]
Looks like it can't access sub-properties.

```
CBaseEntity - func_breakable
- m_iEFlags (Offset 260) (Save)(4 Bytes)
- m_iName (Offset 268) (Save)(4 Bytes)
 Sub-Class Table (1 Deep): m_Collision - CCollisionProperty
 - m_vecMins (Offset 32) (Global|Save)(12 Bytes)
 - m_vecMaxs (Offset 44) (Global|Save)(12 Bytes)
```

$edit_field will only locate "m_iEFlags", "m_iName" and "m_Collision".

**Comment by felis-catus** [comment]
Fixed in 1.10.1.
